src/containers/Home.jsx

```diff
@@ -29,40 +37,62 @@ const Hero = () => (
   </Jumbotron>
 );
 
-const Content = props => (
-  <Container>
-    <Row>
-      <Col sm className="mb-5 mx-3 mx-sm-0">
-        <NewsList
-          numEntries={3}
-          articles={[
-            {
-              title: "Upcoming Elections",
-              description: "Checkout the candidate profiles",
-            },
-            {
-              title: "SESX S01E01",
-              description: "Running a Startup as an SE Student",
-            },
-            {
-              title: "Ibrahim: What I wish I knew in 1A",
-              description: "Intro to the blog",
-            },
-          ]}
-        />
-        <Link to="/news">
-          <Button variant="secondary">See Archive</Button>
-        </Link>
-      </Col>
-      <Col sm className="mb-5 mx-3 mx-sm-0">
-        <EventList events={props.events} />
-        <Link to="/events">
-          <Button variant="secondary">See All</Button>
-        </Link>
-      </Col>
-    </Row>
-  </Container>
-);
+const Content = props => {
+  var upcomingEvent = "";
+  var upcomingEventLink = "";
+  if (props.events[0] && props.events[0].title && props.events[0].link) {
+    upcomingEvent = props.events[0].title;
+    upcomingEventLink = props.events[0].link;
+  }
+  return (
+    <Container>
+      <Row className="mb-sm-5 mx-0 mx-sm-0">
+        <Col
+          sm={{size: "auto", offset: 1}}
+          className="blurb-content order-sm-2"
+        >
+          <h3>Missed us last time? Catch us at: </h3>
+          <p>
+            <a
+              href={upcomingEventLink}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {upcomingEvent}
+            </a>
+            . Or, check out all <Link to="/events"> upcoming events </Link>
```

Comment:
any reason why you're using 2 different types of links?

Reply:
note: upcomingEventLink is an external link while /events is internal 

<Link> is preferred over <a> for internal links b/c it takes absolute paths over <a>’s relative paths. Also if the <link> points to the same page the user is on, it won’t do a full page refresh when clicked. According to: https://stackoverflow.com/questions/43087007/react-link-vs-a-tag-and-arrow-function

Since <Link> is for internal, then <a> is used for external links.